EstateManagement/Factories/ModelFactory.cs

```diff
@@ -120,7 +118,8 @@ public EstateResponse ConvertFrom(Estate estate)
         /// </summary>
         /// <param name="merchant">The merchant.</param>
         /// <returns></returns>
-        public MerchantResponse ConvertFrom(Merchant merchant, MerchantBalance merchantBalance = null)
+        public MerchantResponse ConvertFrom(Merchant merchant,
+                                            MerchantBalance merchantBalance = null)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5881896)